storage-narpc/src/main/java/org/apache/crail/storage/tcp/TcpStorageServer.java

```diff
@@ -167,7 +167,7 @@ public static InetSocketAddress getDataNodeAddress() throws IOException {
 		
 		NetworkInterface netif = NetworkInterface.getByName(ifname);
 		if (netif == null){
-			return null;
+			throw new IOException("Cannot find network interface with name " + TcpStorageConstants.STORAGE_TCP_INTERFACE);
```

Comment:
The exact same code is in RdmaStorageServer and also needs this fix. Can we move this code somewhere to be used by both?